src/test/store/receive-profile.test.js

```diff
@@ -542,21 +550,22 @@ describe('actions/receive-profile', function() {
       const store = blankStore();
       await store.dispatch(retrieveProfileFromStore('FAKEHASH'));
 
-      sinon.assert.calledWithMatch(
-        window.fetch,
-        'https://symbols.mozilla.org/symbolicate/v5',
-        { body: sinon.match(/memoryMap.*libxul/) }
+      expect(window.fetch.mock.calls[1][0]).toBe(
+        'https://symbols.mozilla.org/symbolicate/v5'
       );
+      expect(window.fetch.mock.calls[1][1].body).toMatch(/memoryMap.*libxul/);
```

Comment:
I think you can do:
```js
expect(window.fetch).toHaveBeenLastCalledWith(
  'https://symbols.mozilla.org/symbolicate/v5', { body: /memoryMap.*libxul/ }
);
```
Maybe for the second parameter you'll need `expect.objectContaining({ body: /memoryMap.*libxul/ })`
or `{ body: expect.stringMatching(/memoryMap.*libxul/) }`
or a combination `expect.objectContaining({ body: expect.stringMatching(/memoryMap.*libxul/) })`.
(but please try them in this order so that we have the simplest solution).